<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplicationClass</class>
 <widget class="QMainWindow" name="QtWidgetsApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1438</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>1531</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>用户模式</string>
     </attribute>
     <widget class="QPushButton" name="pb_fous">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>自动对焦</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_open_6">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>多视野采集自动对焦</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_model">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>自动检测</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>请选择相机</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="CameraBox">
             <item>
              <property name="text">
               <string>flir camera</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>haikang camera</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pb_camera">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>保存路径：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtSaveDir">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnSaveDir">
             <property name="text">
              <string>打开...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnSingleImageSave">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>单张图像保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>增    益：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>曝    光：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QSlider" name="sldGain">
             <property name="maximum">
              <number>4799</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="sldExposure">
             <property name="minimum">
              <number>6</number>
             </property>
             <property name="maximum">
              <number>15000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QDoubleSpinBox" name="spinGain">
             <property name="suffix">
              <string>dB</string>
             </property>
             <property name="maximum">
              <double>47.990000000000002</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spinExposure">
             <property name="suffix">
              <string>us</string>
             </property>
             <property name="minimum">
              <double>6.000000000000000</double>
             </property>
             <property name="maximum">
              <double>15000.000000000000000</double>
             </property>
             <property name="value">
              <double>6.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="chkGain">
             <property name="text">
              <string>自动</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkExposure">
             <property name="text">
              <string>自动</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab2">
     <attribute name="title">
      <string>高级模式</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>11</y>
      <width>1431</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGraphicsView" name="imgView"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="TCT_View"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1438</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication.qrc"/>
 </resources>
 <connections/>
</ui>
